components/pipeline-service/staging/stone-stage-p01/deploy.yaml

```diff
@@ -1,2041 +0,0 @@
-apiVersion: v1
```

Comment:
The files were not generated properly, they all seem to be empty :wink:

Reply:
fixed now